dev/run-tests.py

```diff
@@ -255,7 +278,7 @@ def build_spark_sbt(extra_profiles):
 
     print("[info] Building Spark using SBT with these arguments: ", " ".join(profiles_and_goals))
 
-    exec_sbt(profiles_and_goals)
+    exec_sbt(profiles_and_goals, retry=3)
```

Comment:
Would it be better to make `retry` a configurable environment variable with a default value of 3?

Reply:
sounds good！